app/screens/route-details/components/start-ride-button.tsx

```diff
@@ -79,6 +80,17 @@ export const StartRideButton = observer(function StartRideButton(props: StartRid
           analytics().logEvent("start_live_ride_disable_press")
```

Comment:
`_disabled_press`

It's worth adding the disabled reason as a parameter to the event